lib/leap_cli/commands/vagrant.rb

```diff
@@ -138,15 +138,18 @@ def execute(cmd)
   def create_vagrant_file(options)
     lines = []
 
-    basebox = options[:basebox] || 'LEAP/jessie'
+    basebox  = options[:basebox] || 'LEAP/jessie'
```

Comment:
could you please seperate this into a different commit/m.r. ? i don't see how it's related.